tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/konan/tasks/KonanGenerateCMakeTask.kt

```diff
@@ -51,10 +51,10 @@ open class KonanGenerateCMakeTask : DefaultTask() {
     private val host = HostManager.host
 
     private fun generateCMakeLists(
```

Comment:
Do we need that/does it work?

Reply:
It works but it doesn't look like we really need it now.